File: python new work/OOPS/Student.py
class Student:#constructor __init__()
    name:str
    course:str
    degree:str
    def __init__(self,name,course,degree):#method
        self.name=name
        self.course=course
        self.degree=degree
    def display(self):
        print(self.name,self.course,self.degree)
obj1=Student("Anji","Django","MCA")#object
obj1.display()
obj2=Student("Merin","Django","MCA")
obj2.display()

File: JAVASCRIPT WORKS/OOPS/student.js
class Student{

    constructor(name,course,degree){
        this.name=name;
        this.course=course;
        this.degree=degree
    }
    printStudent(){
        console.log(this.name,this.course,this.degree);
    }
}

var obj=new Student("anji","python","mca");
obj.printStudent()